CHANGES.rst

```diff
@@ -7,6 +7,7 @@ Version 0.9.0 (UNRELEASED)
 - Adds a label link to the workflow launcher URL remote origin on the workflow details page.
 - Adds Launch on REANA page allowing the submission of workflows via badge-clicking.
 - Adds 404 Not Found error page.
+- Adds refresh polling to the workflow details page.
```

Comment:
minor: What do you think about `Adds periodic refresh of the workflow details page`?